docs/resources/dms_rocketmq_user.md

```diff
@@ -42,10 +42,10 @@ The following arguments are supported:
 * `instance_id` - (Required, String, ForceNew) Specifies the ID of the rocketMQ instance.
   Changing this parameter will create a new resource.
 
-* `access_key` - (Required, String, ForceNew) Specifies the access key of the user.
+* `name` - (Optional, String, ForceNew) Specifies the name of the user.
```

Comment:
how about we making this `Required` on docs?

Reply:
it has been deleted